db-client-java/src/main/java/com/eventstore/dbclient/PersistentSubscriptionStats.java

```diff
@@ -0,0 +1,121 @@
+package com.eventstore.dbclient;
+
+public class PersistentSubscriptionStats {
+    private double averagePerSecond;
+    private long totalItems;
+    private long countSinceLastMeasurement;
+    private long lastCheckpointedEventRevision;
+    private long lastKnownEventRevision;
+    private Position lastCheckpointedPosition;
+    private Position lastKnownPosition;
```

Comment:
**FIX:** Based on the RFC it should be `LastKnownEventPosition`.
```suggestion
    private Position LastKnownEventPosition;
```

Reply:
Same explanation as above.